packages/serverless-workflow-diagram-editor/package.json

```diff
@@ -30,9 +30,7 @@
   "devDependencies": {
     "@kie-tools/maven-config-setup-helper": "workspace:*",
     "@kie-tools/root-env": "workspace:*",
-    "adm-zip": "^0.5.10",
```

Comment:
Please consider running a `pnpm bootstrap` at the root to update the lock file after the changes in this file.